Example/Backpack/NavigationData.swift

```diff
@@ -0,0 +1,247 @@
+                storyboard.instantiateViewController(withIdentifier: storyboardIdentifier)
+            viewController.modalPresentationStyle = UIModalPresentationStyle.fullScreen
+            return viewController
+        }
+
+        var storyboardName: String
+        var storyboardIdentifier: String
+    }
+
+    // Note - structs are a value type, so if you create a new variable
+    // referencing an existing struct, it will create a unique copy.
+
+    // swiftlint:disable line_length
+
+    static var coloursItem = Item(name: "Colors", result: Storyboard(storyboardName: "Main", storyboardIdentifier: "ColorsViewController"))
+
+    static var gradientsItem = Item(name: "Gradients", result: AppStructure(sections: [Section(name: "", items: [
+        Item(name: "Primary", result: Storyboard(storyboardName: "Gradients", storyboardIdentifier: "PrimaryGradientsViewController")),
+        Item(name: "Baseline scrim", result: Storyboard(storyboardName: "Gradients", storyboardIdentifier: "GradientsViewController"))
+    ])]))
+    static var spacingsItem = Item(name: "Spacings", result: Storyboard(storyboardName: "Main", storyboardIdentifier: "SpacingsViewController"))
+
+    static var radiiItem = Item(name: "Radii", result: Storyboard(storyboardName: "Main", storyboardIdentifier: "RadiiViewController"))
+    static var shadowsItem = Item(name: "Shadows", result: Storyboard(storyboardName: "Main", storyboardIdentifier: "ShadowsViewController"))
+    static var badgesItem = Item(name: "Badges", result: Storyboard(storyboardName: "Badges", storyboardIdentifier: "BadgesViewController"))
+    static var barChartsItem = Item(name: "Bar charts", result: Storyboard(storyboardName: "BarCharts", storyboardIdentifier: "BarChartViewController"))
+    static var bottomSheetItem = Item(name: "Bottom sheet", result: AppStructure(sections: [ Section(name: "", items: [
+        Item(name: "With scroll view", result: Storyboard(storyboardName: "BottomSheet", storyboardIdentifier: "BottomSheetScrollableContentViewController")),
+        Item(name: "With bottom section", result: Storyboard(storyboardName: "BottomSheet", storyboardIdentifier: "BottomSheetScrollableContentViewController")),
+        Item(name: "Presenting another sheet", result: Storyboard(storyboardName: "BottomSheet", storyboardIdentifier: "BottomSheetScrollableContentViewController")),
+        Item(name: "Non-scrollable content", result: Storyboard(storyboardName: "BottomSheet", storyboardIdentifier: "BottomSheetScrollableContentViewController")),
+        Item(name: "Resizeable content height", result: Storyboard(storyboardName: "BottomSheet", storyboardIdentifier: "BottomSheetScrollableContentViewController"))
+    ])]))
+    static var buttonsItem = Item(name: "Buttons", result: AppStructure(sections: [Section(name: "", items: [
+        Item(name: "Primary", result: Storyboard(storyboardName: "Buttons", storyboardIdentifier: "ButtonViewController")),
```

Comment:
There's a lot of repition in these declarations, is that something we could improve?